3Sum.py

```diff
@@ -0,0 +1,26 @@
+// Time Complexity : O(nlogn)
+// Space Complexity : O(n)
+// Did this code successfully run on Leetcode : Yes
+// // Any problem you faced while coding this : No
+
+// Your code here along with comments explaining your approach
+
+class Solution:
+    def threeSum(self, nums: List[int]) -> List[List[int]]:
+        nums.sort()
+        ans=set()
```

Comment:
Good use of set to avoid checking duplicates. Notice that your space complexity has increased.